MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillImpl.kt

```diff
@@ -87,7 +112,7 @@ internal class AddressAutofillImpl(
             accessToken: String,
             app: Application,
             locationEngine: LocationEngine,
-        ): AddressAutofillImpl {
+       ): AddressAutofillImpl {
```

Comment:
nit looks like formatting is not correct here

Reply:
No, it's actually now correct but there was an extra space before :)